File: c/src/trabalho.c
#include<stdio.h>
#include<math.h>

int main() {
	int vetor[10], i, num[10];
	
	for(i = 0; i < 10; i++){
		printf("Digite o %d* numero:\n", (i + 1));
		scanf("%d", & num[i]);
		vetor[i] = pow(num[i], 3);
	}
	
	printf("\n\n");
	
	for(i = 0; i < 10; i++){
		printf("%d ^ 3 = %d\n", num[i], vetor[i]);
	}
}

File: php/src/trabalho.php
<?php

$funcao = $_POST["action"];

if (function_exists($funcao)) {
    call_user_func($funcao);
}

function calc()
{
    $num =  explode(',', $_POST['vet']);
    
    for ($i = 0; $i < 10; $i++) { 
        $vetor[$i] = pow($num[$i], 3);
    }

    for ($i = 0; $i < 10; $i++) { 
        echo "<script>localStorage.setItem('vetor[$i]', '$num[$i] ^ 3 = $vetor[$i]');</script>";
    }
    
    echo "<script>location.href = 'index.html';</script>";
}
